sap-schemas/v3.0/README.md

```diff
@@ -172,6 +172,67 @@ Example:
 }
 ```
 
+## Operation level extensions
+
+### `x-sap-operation-kind`
+
+- Type: `String`
+- Used at: [Operation Object](https://spec.openapis.org/oas/v3.0.3#operation-object)
+- Description: The intent of the operation, a fixed list of values that can be extended on request
+
+| Intent            | Description                                                                                                           | HTTP verbs     |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------- | -------------- |
+| `create-single`   | insert a single object into a collection                                                                              | `POST`         |
+| `create-multiple` | insert multiple objects into a collection                                                                             | `POST`         |
+| `read-single`     | read a single object                                                                                                  | `GET`          |
+| `read-collection` | read a collection, can typically be combined with query options `$search`, `$filter`, `$orderby`, `$top`, and `$skip` | `GET`          |
+| `update-single`   | full or partial update of a single object, may allow "deep" update of a root object and its nested objects            | `PATCH`, `PUT` |
```

Comment:
Do we want to distinguish between "shallow" and "deep" update, or is this rather expressed by the request payload structure which has to allow nested objects for "deep" updates?